backend/controllers/posts.controller.js

```
@@ -0,0 +1,66 @@
+      return res.status(403).json({ error: 'Permission denied' });
+    }
+
+    const { name, position, salary, experience, location, skills, deadline } = req.body;
+    validateJobPostData({ name, position, salary, experience, location, skills });
+
+    const skillIds = await upsertSkills(skills);
+
+    const post = new Post({
+      name,
+      position,
+      salary: parseFloat(salary),
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)